scijava-legacy/pom.xml

```diff
@@ -147,7 +145,11 @@
 			<groupId>org.scijava</groupId>
 			<artifactId>scijava-common3</artifactId>
 			<version>${project.version}</version>
-			<scope>test</scope>
+			<!--
```

Comment:
Huh, I thought this was not necessary at compile time, but only necessary at test compile time.

Reply:
With the scijava-common3 declaration completely missing, everything works, but `dependency:analyze` warns that common3 is "used but undeclared" in the tests. So the natural fix is to add the direct dependency at test scope. But if you do that, it overrides the transitive compile scope coming in from scijava-ops-api and probably others, and the compilation fails due to the org.scijava.common3 module being missing. Hence the XML comment explaining the situation.

This is an edge case around Maven's dependency resolution infrastructure, I suppose. :shrug: